README.md

```diff
@@ -128,6 +128,22 @@ $addon         = $api->addon->fetch('ao_8nDvQYYGQI5o4H')->delete();
 $settlement    = $api->settlement->fetch('setl_7IZKKI4Pnt2kEe');
 $settlements   = $api->settlement->all();
 $reports       = $api->settlement->reports(array('year' => 2018, 'month' => 2));
+
+//Paper NACH
```

Comment:
@HancilSequeira , this much detailed example must go inside example folder. //cc @chirag-patel-rp